README.md

```diff
@@ -129,3 +129,43 @@ SimpleCov.start
 ```
 
 You can see your test coverage when you run your tests. If you want this in a graphical form, uncomment the `HTMLFormatter` line and see what happens!
+
+My notes
+--------
+
+How to use
+----------
+Fork and clone this repo
+
+Run bundle install
+
+
+```CREATE DATABASE peep_list```
+```CREATE TABLE peep_list(id SERIAL PRIMARY KEY, peep VARCHAR(180));```
```

Comment:
Maybe is a good idea to refer user to the files in folder db/migrations when setting up the database?